cloud_governance/zombie_cluster/delete_iam_resources.py

```diff
@@ -0,0 +1,55 @@
+from cloud_governance.common.logger.init_logger import logger
+
+
+class DeleteIAMResources:
+    """
+    This class deleted the IAM resources
+    IAM Role
+    IAM User
+    """
+
+    def __init__(self, iam_client):
+        self.iam_client = iam_client
+
+    def delete_iam_zombie_resource(self, resource_id: str, resource_type: str):
```

Comment:
Add documentations
@logger_time_stamp
@typeguard.typechecked

Reply:
Do we need @logger_time_stamp, is it safe ?

Reply:
I think it's better to delete it, because expose the id of the resource.

Reply:
I think it's better to delete, because it expose id of the resource.